src/status_im/ui/components/list_item/styles.cljs

```diff
@@ -13,7 +13,7 @@
             {:line-height 22 :font-size 15}
             {:line-height 20 :font-size 17})
     subtitle
-    (assoc :font-weight 500 :font-size 15)))
+    (assoc :font-weight "500" :font-size 15)))
```

Comment:
Any reason why?

Reply:
idk why but in current version of app it crashes in run time

Reply:
it takes an enum from `('normal', 'bold', '100', '200', '300', '400', '500', '600', '700', '800', '900')`, passing a number won't work